docs/spec/src/integration/spec/3-data-structs.md

```diff
@@ -126,7 +126,57 @@ Users who want to pay-per-blob need to set the cumulative_payment. `timestamp` i
 
 **NOTE:** There will be a lot of subtleties added to this logic with the new separate-payment-per-quorum model that is actively being worked on.
 
-An RPC call to the Disperser’s `GetPaymentState` method can be made to query the current state of an `account_id`. A client can query for this information on startup, cache it, and then update it manually when making dispersals. In this way, it can keep track of its reserved bandwidth usage and current cumulative_payment and set them correctly for subsequent dispersals.
+An RPC call to the Disperser's `GetPaymentState` method can be made to query the current state of an `account_id`. A client can query for this information on startup, cache it, and then update it manually when making dispersals. In this way, it can keep track of its reserved bandwidth usage and current cumulative_payment and set them correctly for subsequent dispersals.
+
+### BlobKey (Blob Header Hash)
+
+The `blobKey` (also known as `blob_header_hash` or `blobHeaderHash`) serves as the _primary lookup key_ throughout the EigenDA system. It uniquely identifies a blob dispersal and is used for querying dispersal status, retrieving blobs from the network, and linking blobs to their certificates. The `blobKey` is computed as the keccak256 hash of the ABI-encoded `BlobHeader`, and is cryptographically equivalent to the `blob_header_hash` used in on-chain verification.
+
+#### Computing the BlobKey
+
+The hashing follows a nested structure where the core dispersal fields are hashed first, then combined with the payment metadata hash:
+
+```solidity
+blobKey = keccak256(
+    abi.encode(
+        keccak256(abi.encode(blobHeader.version, blobHeader.quorumNumbers, blobHeader.commitment)),
+        blobHeader.paymentHeaderHash
+    )
+)
+```
+
+This two-level approach reflects the separation between dispersal parameters and payment concerns. When a rollup receives an encoded DA commitment from the proxy, the `blobKey` can be extracted by deserializing the BlobCertificate from the commitment payload, extracting its BlobHeader, and computing the hash as shown above.
+
+#### Relationship to Other Data Structures
+
+The diagram below illustrates how `blobKey` relates to other core EigenDA data structures:
+
+![Data Structure Relationships](../../assets/integration/blob-key-relationships.png)
```

Comment:
why are we duplicating a diagram that already exists with an additional commit link that a user could never reference?
https://layr-labs.github.io/eigenda/integration/spec/3-data-structs.html#eigenda-certificate-dacert

Reply:
remove the duplicated image: [ba6ef14](https://github.com/Layr-Labs/eigenda/pull/2234/commits/ba6ef14024ae17ffec120e4b7066608cab2e38b7)